runners/portability/java/src/main/java/org/apache/beam/runners/portability/PortableRunner.java

```diff
@@ -200,11 +143,44 @@ public PipelineResult run(Pipeline pipeline) {
           prepareJobResponse.getArtifactStagingEndpoint();
       String stagingSessionToken = prepareJobResponse.getStagingSessionToken();
 
+      ImmutableList.Builder<StagedFile> filesToStageBuilder = ImmutableList.builder();
```

Comment:
Perhaps it's worth pulling this out into a separate method?

Reply:
done.